MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -32,6 +32,7 @@ import Foundation
     var authTokens: MSALNativeAuthTokens
     let configuration: MSALNativeAuthConfiguration
     private let cacheAccessor: MSALNativeAuthCacheInterface
+    private let application: MSALNativeAuthPublicClientApplication?
```

Comment:
should this reference be weak?

Reply:
This should be a strong reference, in order to hold a the public client application and have access to the get access token method, in case the user nullify the original NativeAuthPublicClientApplication object. This property is filled with a copy of the object passed in its constructor (line 53).